### ISSUE TITLE
0.2.4 is broken by hash tentative

### ISSUE THREAD

**Issue by flavien-sindou** [impocode/jsonapi-pydantic#4]
Executing the 1st example from the Readme with version 0.2.4 returns the following error : 

https://github.com/impocode/jsonapi-pydantic?tab=readme-ov-file#usage

```
top_level = TopLevel(**external_data)
.venv/lib/python3.10/site-packages/jsonapi_pydantic/v1_0/toplevel.py:55: in check_all_values
    hashes = {d._hash for d in data["data"]}
.venv/lib/python3.10/site-packages/jsonapi_pydantic/v1_0/toplevel.py:55: in <setcomp>
    hashes = {d._hash for d in data["data"]}
E   AttributeError: 'dict' object has no attribute '_hash'
```

I revert the version to 0.2.3 and it's working fine.

This commit seems to be responsible : f2b4ecba768d0e3524dd5b150c61a4458c8e6092

**Comment by impocode** [comment]
Hello @flavien-sindou ! Thank you for your issue. Yes, I think I really broke it in version `0.2.4`. I'll try to fix it soon.

**Comment by impocode** [comment]
Hello @flavien-sindou ! I fixed this in the new version `0.2.5`.
